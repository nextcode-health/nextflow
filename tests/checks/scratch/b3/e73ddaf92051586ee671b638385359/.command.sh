#!/bin/bash -ue
echo beta memry: 1 GB
echo beta queue: small
