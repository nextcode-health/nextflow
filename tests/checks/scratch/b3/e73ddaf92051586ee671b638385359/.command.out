beta memry: 1 GB
beta queue: small
